scripts/linux/waku.service

```diff
@@ -0,0 +1,44 @@
+# To configure the service, use `systemctl edit waku.service`
+# and override the environment variables in this file
+#
+# To completely override the start command, override the `ExecStart` value
+# instead by first emptying it, then specifying a new one:
+#
+# [Service]
+# ExecStart=
+# ExecStart=/usr/bin/waku --cluster-id=1
+
+[Unit]
+Description=Waku
+Wants=network-online.target
+After=network-online.target
+
+[Install]
+WantedBy=multi-user.target
+
+[Service]
+# TWN
+Environment=CLUSTER_ID=1
+
+# Default ports
+Environment=TCP_PORT=9000
+Environment=REST_ENABLED=true
+Environment=REST_PORT=8645
+Environment=METRICS_ENABLED=true
+Environment=METRICS_PORT=8008
+
+# Default group = waku
+WorkingDirectory=/var/lib/waku
+TimeoutSec=1200
+Restart=always
+
+# List of non-recoverable error codes
+RestartPreventExitStatus=166
+
+ExecStart=/usr/bin/waku \
+  --cluster-id=${CLUSTER_ID} \
+  --port=${TCP_PORT} \
+  --rest=${REST_ENABLED} \
+  --rest-port=${REST_PORT} \
+  --metrics=${METRICS_ENABLED} \
+  --metrics-port=${METRICS_PORT}
```

Comment:
@chaitanyaprem I'm not sure what flags to use here by default. Should I use the same from go-waku-compose ?

Reply:
go-waku-compose is meant for TWN so cluster-id is mentioned as 1. 
But for status since they use static sharding cluster-id is 16. So maybe we can indicate that as default and add a note to change it accordingly. 

Wrt rest, we better leave it enabled as it is useful to query info from a node. 
Metrics also should be better enabled by default. 

Wrt rest and metrics ports, I am wondering if it is better to not specify and leave them as default. Since this is required by Infra team, maybe better to check with them.

Reply:
@jakubgs what would you like to see in the systemd template for go-waku?

Reply:
You can use our docker compose as a starting point for what flags to include:
https://github.com/status-im/infra-role-nim-waku/blob/master/templates/docker-compose.yml.j2

As for systemd service config, you can take a look at our beacon node one:
https://github.com/status-im/infra-role-beacon-node-linux/blob/master/templates/beacon-node.service.j2

And include fields like `Documentation`, `Restart`, and `EnvironmentFile`.
With `EnvironmentFile` you could put the defaults under `/etc` and allow the user to easily modify them.